src/node_options.cc

```diff
@@ -128,9 +128,23 @@ void EnvironmentOptions::CheckOptions(std::vector<std::string>* errors) {
   }
 
   if (!es_module_specifier_resolution.empty()) {
-    if (es_module_specifier_resolution != "node" &&
-        es_module_specifier_resolution != "explicit") {
-      errors->push_back("invalid value for --es-module-specifier-resolution");
+    if (!experimental_specifier_resolution.empty()) {
```

Comment:
Honestly I found this logic here to be a bit spaghetti... anyone have concerns or ideas on how to clean it up a bit?

Reply:
Not concerned about it since it doesn't cross encapsulation boundaries. Three branches for three scenarios is expected.